src/defaultTo.js

```diff
@@ -19,7 +19,8 @@ var _curry2 = require('./internal/_curry2');
  *      defaultTo42(null);  //=> 42
  *      defaultTo42(undefined);  //=> 42
  *      defaultTo42('Ramda');  //=> 'Ramda'
+ *      defaultTo42(parseInt('string')); //=> 42
  */
 module.exports = _curry2(function defaultTo(d, v) {
-  return v == null ? d : v;
+  return v == null || v !== v ? d : v;
```

Comment:
I'm fine with this. The other option is to encapsulate the `v !== v` logic in an internal `_isNaN` function.